app/controllers/candidate_interface/gcse/missing_qualification_controller.rb

```diff
@@ -0,0 +1,91 @@
+module CandidateInterface
+  class Gcse::MissingQualificationController < Gcse::BaseController
+    include Gcse::ResolveGcseEditPathConcern
+    before_action :redirect_to_application_form_if_current_qualification_missing, only: %i[edit update]
+
+    def new
+      @gcse_missing_form = if current_qualification
```

Comment:
there should always be a current_qual here otherwise the before action ☝️ will redirect them back to the application form